inference-engine/src/inference_engine/CMakeLists.txt

```diff
@@ -160,7 +161,7 @@ if (TBBBIND_2_4_FOUND)
 endif()
 
 target_link_libraries(${TARGET_NAME} PRIVATE pugixml openvino::itt ${CMAKE_DL_LIBS} Threads::Threads
-                                             ${NGRAPH_LIBRARIES} inference_engine_transformations)
+                                             ${NGRAPH_LIBRARIES} ${FRONTEND_LIBRARIES} inference_engine_transformations)
```

Comment:
use `frontend_manager` directly.

Reply:
please, also add `frontend_manager` to `inference_engine_s` instead of adding this dependency to test utils, etc